plugins/hwp-previews/package.json

```diff
@@ -1,6 +1,7 @@
 {
-	"name": "hwp-previews",
-	"version": "0.0.1",
+	"name": "@hwp-previews/wordpress-plugin",
```

Comment:
We don't own this on npm and even if its private, we want to use the @wpengine namespace just in case

Reply:
Thanks @theodesp I will update the namespace. Thanks for spotting this.